go-app.js

```diff
@@ -127,6 +127,21 @@ go.app = function() {
         choices: self.im.config.localities.map(function (locality) {
           return new Choice(locality, locality);
         }),
+        next: 'Landmark_Entry'
+      });
+    });
+
+    self.states.add('Landmark_Entry', function(name) {
+      var question = $("What is the closest landmark for the patient?");
+      var error = $("Please enter a landmark");
+      return new FreeText(name, {
+        question: question,
+        check: function(content) {
+          if (content.length === 0) {
+            return error;
+          }
```

Comment:
Updated to this:

```
    self.states.add('Landmark_Entry', function(name) {
      var question = $("What is the closest landmark for the patient?");
      var error = $("Please enter a landmark");
      return new FreeText(name, {
        question: question,
        next: 'ID_Type_Entry'
      });
    });

```

jslint then moans about error not being used.

```
Running "jshint:all" (jshint) task

   src/app.js
     89 |      var error = $("Please enter a landmark");
                   ^ 'error' is defined but never used.

>> 1 error in 7 files

```

How should this be done please?